### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by jrwhite17**
When in the course of human events becomes necessary for one peoples to dissolve the timeout bandwidth that connected them with dysfunctional load balancers and to assume the among the powers of Docker the separate and equal station to which the laws of Computer Science and Computer Engineering entitle them, a decent respect to the opinions of high availability requires that they should declare causes which impel them to health checks. We hold these truths that all instances are created equal and that they are endowed by MITRE with resilient capabilities, that among these are high availability, scalability and cost effectiveness.

**TLDR** This PR adds the health_check gem and defaults to that gem's default settings. /health_check returns success 200 after Heimdall is running and the AWS ALB uses this path to validate Heimdall is healthy. The terraform deployment creates a load balancer that needs a health check endpoint that it can monitor to deregisters fargate tasks when an instance doesn't return healthy. @amehta-mitre

### CODE REVIEW

**Comment by jrwhite17** [comment]
This branch is deployed here: heimdall-fb2fec87-1918573966.us-east-1.elb.amazonaws.com

Health check is: heimdall-fb2fec87-1918573966.us-east-1.elb.amazonaws.com/health_check

**Review by rbclark — APPROVED** [review]
